src/GeneralStateTestsFiller/stEWASMTests/overwriteDataFiller.yml

```diff
@@ -0,0 +1,55 @@
+# create contract from transaction overwrite previous data
+---
+overwriteData:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    # tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: ''
+      storage: {}
+  expect:
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value: !!int -1
+      network:
+        - "Byzantium"
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          balance: '99999929052'
+        # newly-created contract
+        6295ee1b4f6dd65047762f924ecd367c17eabf8f:
+          code: '0x0061736d01000000010401600000030201000503010001071102066d656d6f72790200046d61696e00000a05010300010b'
+  transaction:
+    data:
+    - |
+      (module
+        (import "ethereum" "finish" (func $finish (param i32 i32)))
+        (memory 100)
```

Comment:
Please don't use memory 100.

Reply:
@axic why not? The reason I'm asking is that this tx breaks the sync with wagon so I'm wondering if you could pinpoint a problem.

Reply:
We have (should have?) specific tests for high memory usage. Other tests which do not need that should not use it.

Reply:
@axic fixed